os/android/libhwcservice/test/hwcservice_test.cpp

```diff
@@ -61,8 +67,16 @@ int main(int argc, char** argv) {
   bool disable_hdcp_for_display = false;
   bool disable_hdcp_for_all_display = false;
   bool restore = false;
+#ifdef ENABLE_PANORAMA
+  bool enable_panorama = false;
```

Comment:
Okay, will change the variable names accordingly. Prefer to keep the 2 variables since the test app is not long last daemon and no way to keep current panorama status. Using separated variable will make the code simpler.